negotiation/neural/model_builder.py

```diff
@@ -180,26 +183,26 @@ def make_base_model(model_opt, mappings, gpu, checkpoint=None):
     # Make encoder.
     src_dict = mappings['vocab']
     src_embeddings = make_embeddings(model_opt, src_dict)
-
     encoder = make_encoder(model_opt, src_embeddings)
-
+    # Make context embedder.
+    if model_opt.num_context > 1:
+      original_enc_type = model_opt.encoder_type
+      model_opt.encoder_type = "mean"
```

Comment:
this is a little weird... doesn't changing encoder_type affect later make_embedding? ideally we should have a different arg like "context encoder type" or something